sechub-testframework/src/test/java/com/mercedesbenz/sechub/test/JUnitAssertionAddonTest.java

```diff
@@ -0,0 +1,35 @@
+package com.mercedesbenz.sechub.test;
```

Comment:
SPDX header missing

Reply:
Not only the test but also the tested class did not contain the SPDX header. I added now both.

By the way: You do not longer need to comment those missing SPDX headers, because our release build does now automatically create a PR at the release, when at least one SPDX header is missing. The release documentation does mention this and when we build a release and such a PR happens, the person creating the release has just to apply the PR merge and all is done.